ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -313,6 +314,7 @@ export default class DocumentMetadata extends Component {
 
   render() {
     const document = this.props.documentDetails;
+    // TODO: clear the read access thing
```

Comment:
Create an issue for the todo?

Reply:
What does this exactly mean? `clear the read access thing`?